test/stack-dependencies.txt

```diff
@@ -43,6 +43,8 @@ conduit-extra 1.3.5
 connection 0.3.1
 containers 0.6.2.1
 contravariant 1.5.2
+control-bool 0.2.1
```

Comment:
`Conversion` has a dependence on `control-bool` where it only uses one function from it. Will remove it a future PR